public/_headers

```diff
@@ -4,9 +4,9 @@
   Content-Security-Policy: frame-ancestors 'self';
   Content-Security-Policy: object-src 'none'; 
   Content-Security-Policy: base-uri 'self'; 
-  Content-Security-Policy: img-src * 'self' https://www.google-analytics.com;
+  Content-Security-Policy: img-src 'self' https://www.google-analytics.com data:;
   Content-Security-Policy: manifest-src 'self' https://*.lunie.io https://*.netlify.app;
-  Content-Security-Policy: script-src-elem * 'unsafe-inline';
+  Content-Security-Policy: script-src-elem 'unsafe-inline' data:;
```

Comment:
`data:` is considered dangerous I think lets investigate and maybe change

Reply:
not so easy to set these headers. That is for sure